diff -Naur open-gpu-kernel-modules-575.51.02.old/utils.mk open-gpu-kernel-modules-575.51.02/utils.mk
--- open-gpu-kernel-modules-575.51.02.old/utils.mk	2025-04-17 18:39:22.355962583 +0200
+++ open-gpu-kernel-modules-575.51.02/utils.mk	2025-04-17 18:42:46.397686515 +0200
@@ -116,9 +116,6 @@
 XZ                    ?= xz
 PKG_CONFIG            ?= pkg-config
 
-NV_BUILD_USER         ?= $(shell whoami)
-NV_BUILD_HOST         ?= $(shell hostname)
-
 NV_AUTO_DEPEND        ?= 1
 NV_VERBOSE            ?= 0
 
@@ -550,7 +547,7 @@
   ifneq ($$(NVIDIA_NVID_EXTRA),)
     $(1)_BUILD_NVID := $$($$(strip $(1))_BUILD_NVID)  $$(NVIDIA_NVID_EXTRA)
   endif
-  $(1)_BUILD_NVID := $$($$(strip $(1))_BUILD_NVID)  ($$(NV_BUILD_USER)@$$(NV_BUILD_HOST))
+  $(1)_BUILD_NVID := $$($$(strip $(1))_BUILD_NVID)
   # g_nvid_string.c depends on all objects except g_nvid_string.o, and version.mk
   $(NVIDSTRING): $$(filter-out $$(call BUILD_OBJECT_LIST,$$(NVIDSTRING)), $(3)) $$(VERSION_MK)
 	$(at_if_quiet)$$(MKDIR) $$(dir $$@)
